Automattic/jetpack
# My Jetpack: Update Boost product icon

Updates the Boost product icon 

<img width="271" alt="image" src="https://user-images.githubusercontent.com/746152/159328969-7ae0ad89-af5d-406a-b9e2-e2868d5031c1.png">


Fixes #23515

#### Changes proposed in this Pull Request:

* Uses the new chart-like icon

#### Jetpack product discussion
p1HpG7-fbl-p2

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:

* Checkout this branch
* Visit My Jetpack and confirm the Boost cards shows the new icon
